console/console.c

```diff
@@ -910,11 +910,19 @@ ExecCmd(void)
     /* setup new process with clean file descriptors
      * stderr still goes to stderr...so user sees it
      */
+#ifdef HAVE_CLOSEFROM
+    for (i = 3; i <= pout[0] || i <= pin[1]; i++) {
+	if (i != pout[0] && i != pin[1])
+	    close(i);
+    }
+    closefrom(i);
+#else
     i = GetMaxFiles();
     for ( /* i above */ ; --i > 3;) {
```

Comment:
Should this (existing condition) be `--i > 2`?

Reply:
Yeah, looks like!  I'll patch that right now.  Thanks! (for this and all the rest)